### DISCUSSION TITLE
unknown compound in Saudi Arabia GP

### DISCUSSION THREAD

**Discussion by markste-in** [theOehrly/Fast-F1#137] [Q&A]
I just loaded all the data from the SA GP and checked the compound

<img width="454" alt="image" src="https://user-images.githubusercontent.com/33337590/160496206-61a7dc42-3962-4944-933b-483cd8daa66a.png">

Somehow there is the "UNKNOWN" compound again in the data. Do we know if UNKOWN maps to HARD again (like in the Bahrain GP?)

**Comment by theOehrly** [comment]
I'm not sure what "UNKNOWN" is here. It might be interesting to compare the relevant laps to the data in the official app and to tv footage. But that's quite a bit of effort.
